integreat_cms/static/src/js/pages/page-clickable-sidebar.ts

```diff
@@ -0,0 +1,23 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const boxes = document.querySelectorAll(".sidebar-box");
+    const cookies = document.cookie.split("; ").map((cookie) => cookie.split("="));
+    for (let i = 0; i < boxes.length; i++) {
```

Comment:
The for-loop with a counter seems a bit old fashioned to me, I guess a `forEach` would be the more modern approach?
```suggestion
    boxes.forEach((box) => {
```